WebDriverAgentMac/WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -234,4 +241,30 @@ + (NSDictionary *)currentCapabilities
   };
 }
 
++ (NSArray<NSString *> *)appArgumentsForLocale:(NSDictionary<NSString *, NSString *> *)appLocale
+{
+  // https://developer.apple.com/forums/thread/678634
+  NSMutableArray<NSString *> *result = [NSMutableArray new];
+  if (nil != appLocale[@"language"]) {
+    [result addObject:@"-AppleLanguages"];
+    [result addObject:[NSString stringWithFormat:@"(%@)", appLocale[@"language"]]];
+  }
+  if (nil != appLocale[@"locale"]) {
+    [result addObject:@"-AppleLocale"];
+    [result addObject:[NSString stringWithFormat:@"%@", appLocale[@"locale"]]];
+  }
+  if (nil != appLocale[@"useMetricUnits"]) {
+    [result addObject:@"-AppleMetricUnits"];
+    [result addObject:[NSString stringWithFormat:@"<%@/>", appLocale[@"useMetricUnits"] ? @"true" : @"false"]];
+  }
+  if (nil != appLocale[@"measurementUnits"]) {
+    [result addObject:@"-AppleMeasurementUnits"];
```

Comment:
May `AppleTemperatureUnit` also work?
https://github.com/apple/swift-corelibs-foundation/blob/7f62c8c52aeed9a86b503fe7b662c48eda5d0f8a/CoreFoundation/Locale.subproj/CFLocale.c#L1593C48-L1593C68

Reply:
maybe, not sure how to test that though...

Reply:
feel free to add it if you know